### ISSUE TITLE
Tuning never finishes on Intel HD Graphics 620 / Windows 10

### ISSUE THREAD

**Issue by alreadydone** [gcp/leela-zero#2090]
https://github.com/featurecat/lizzie/issues/465#issuecomment-448707555
https://github.com/featurecat/lizzie/issues/465#issuecomment-449476509

The first 6 configs come out quickly but the tuning run is not ended after 1 hour (with task manager showing it's still compute-bound). The cpuonly version works fine on the machine.
https://github.com/featurecat/lizzie/issues/465#issuecomment-448711813

**Comment by foreverpokemongo** [comment]
I can confirm similar behavior on computers with similar hardware configuration. I copy the output of leela below:

AutoGTP v17
Using 1 thread(s) for GPU(s).
Starting tuning process, please wait...
Net filename: networks/939ebbcc73423bba129656a68d9c0f9e0db88596619487dee7b4abd159e781fa.gz
net: 939ebbcc73423bba129656a68d9c0f9e0db88596619487dee7b4abd159e781fa.
./leelaz --tune-only -w networks/939ebbcc73423bba129656a68d9c0f9e0db88596619487dee7b4abd159e781fa.gz
Leela Zero 0.16  Copyright (C) 2017-2018  Gian-Carlo Pascutto and contributors
This program comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it
under certain conditions; see the COPYING file for details.

Using 2 thread(s).
RNG seed: 7300609878876702653
BLAS Core: Haswell
Detecting residual layers...v1...256 channels...40 blocks.
Initializing OpenCL (autodetecting precision).
Detected 1 OpenCL platforms.
Platform version: OpenCL 2.1
Platform profile: FULL_PROFILE
Platform name:    Intel(R) OpenCL
Platform vendor:  Intel(R) Corporation
Device ID:     0
Device name:   Intel(R) HD Graphics 530
Device type:   GPU
Device vendor: Intel(R) Corporation
Device driver: 23.20.16.4973
Device speed:  1150 MHz
Device cores:  24 CU
Device score:  621
Device ID:     1
Device name:   Intel(R) Core(TM) i5-6600 CPU @ 3.30GHz
Device type:   CPU
Device vendor: Intel(R) Corporation
Device driver: 7.6.0.611
Device speed:  3300 MHz
Device cores:  4 CU
Device score:  521
Selected platform: Intel(R) OpenCL
Selected device: Intel(R) HD Graphics 530
with OpenCL 2.1 capability.
Half precision compute support: Yes.
OpenCL: using fp16/half compute support.

Started OpenCL SGEMM tuner.
Will try 290 valid configurations.
(1/290) KWG=16 KWI=2 MDIMA=8 MDIMC=8 MWG=16 NDIMB=8 NDIMC=8 NWG=16 SA=1 SB=1 STRM=0 STRN=0 VWM=2 VWN=2 1.3833 ms (85.3 GFLOPS)
(3/290) KWG=16 KWI=2 MDIMA=8 MDIMC=8 MWG=64 NDIMB=8 NDIMC=8 NWG=16 SA=1 SB=1 STRM=0 STRN=0 VWM=2 VWN=2 1.3139 ms (89.8 GFLOPS)
(5/290) KWG=16 KWI=2 MDIMA=8 MDIMC=8 MWG=32 NDIMB=8 NDIMC=8 NWG=32 SA=1 SB=1 STRM=0 STRN=0 VWM=2 VWN=2 0.8000 ms (147.4 GFLOPS)
(6/290) KWG=16 KWI=2 MDIMA=8 MDIMC=8 MWG=64 NDIMB=8 NDIMC=8 NWG=32 SA=1 SB=1 STRM=0 STRN=0 VWM=2 VWN=2 0.7863 ms (150.0 GFLOPS)
(14/290) KWG=32 KWI=2 MDIMA=8 MDIMC=8 MWG=32 NDIMB=8 NDIMC=8 NWG=32 SA=1 SB=1 STRM=0 STRN=0 VWM=2 VWN=2 0.7496 ms (157.4 GFLOPS)

Basically I can only proceed one step further than the guy reported in lizzie repositary, but nothing else.
